README.md

```diff
@@ -43,59 +58,81 @@ After your Lambda@Edge environment is prepared, install the starter kit:
 
-1. Provision the function with [Lambda@Edge permissions](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-edge-permissions.html) and [associate the function](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/associate-function.html) with your CloudFront distribution. Set the CloudFront trigger for this function to be `Viewer Request`.
+1. Provision the function with [Lambda@Edge permissions](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/lambda-edge-permissions.html) and [associate the function](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/associate-function.html) with your CloudFront distribution.
 
    > Note: CloudFront triggers are associated with only one specific version of your Lambda function. Remember to update the CloudFront trigger assignment as needed when pushing new versions of your Lambda function. For example, you may need one function that handles receiving viewer requests (viewer request trigger) and one function that handles returning a response to the viewer (viewer response trigger).
 
 1. Test your Lambda@Edge function. It should return a simple home page with the results of your feature flag test and User ID cookie.
 
-   > You can find details on how the starter kit works in the CloudWatch console under **Logs > Log groups > /aws/lambda/<YOUR_LAMBDA_NAME>**. Click your Lambda's log group to follow the entire process of reading the headers, assigning the User ID, fetching the client configuration, and getting a variation key for a particular User ID using the Kameleoon NodeJS SDK.
+   > You can find details on how the starter kit works in the CloudWatch console under **Logs > Log groups > /aws/lambda/<YOUR_LAMBDA_NAME>**. Click your Lambda's log group to follow the entire process.
 
 1. Adjust your lambda's configuration as needed. For example, you may need to increase your function's memory, storage, and timeout threshold to accommodate your needs.
 
 You can now use Kameleoon's feature flagging and experimentation as desired. You can modify the cookies and headers based on experimentation results, add hooks to the **Origin Request** and **Origin Response** CloudFront triggers to perform origin redirects or dynamic asset manipulation, or add more services to the pipeline including your own logging systems, databases, CDN origins, and more. Keep in mind that Lambda@Edge has some limitations, which you can familiarize yourself with in the [Edge Functions Restrictions](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/edge-functions-restrictions.html) article.
 
+## Development
+
+### Testing Locally
+
+To avoid the need to upload your Lambda function to AWS every time you make a change, you can test your Lambda function locally using the `AWS SAM` runtime interface client.
```

Comment:
To avoid uploading your Lambda function to AWS every time you make a change, ...